Xamarin.Movies/Xamarin.Movies/Converters/MovieDbImageConverter.cs

```diff
@@ -19,8 +19,10 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
         {
             var imageName = value as string;
             if (imageName == null) return string.Empty;
-
-            return ImageSource.FromUri(new Uri($"{internalSettings.MovieDbImageBaseUrl}/{imageName}"));
+            var uri = new Uri($"{internalSettings.MovieDbImageBaseUrl}{imageName}");
+            var image= ImageSource.FromUri(uri);
+            
+            return image;
```

Comment:
This really works well, but I suppose this is something related in the way how Xamarin Forms handles the Converters, I don't see an actual difference between create and return the Image Source, actually It always works with  brand new references, no state held between `Convert` method calls.